Meta/run.py

```diff
@@ -741,13 +760,15 @@ def set_up_machine_devices(config: Configuration):
             config.qemu_cpu = None
             return
 
-    elif config.architecture == Arch.RISCV64:
+    elif config.architecture == Arch.Aarch64 or config.architecture == Arch.RISCV64:
         config.qemu_machine = "virt"
         config.cpu_count = None
+        config.vga_type = None
+        config.display_backend = None
```

Comment:
Is this the windows qemu quirk from above? Otherwise it would be nice if our normal display backend selection worked.

Reply:
I removed these lines, they seem to still work.